Admin/AbstractAdmin.php

```diff
@@ -2017,7 +2017,7 @@ public function buildBreadcrumbs($action, MenuItemInterface $menu = null)
             E_USER_DEPRECATED
         );
         if ($this->breadcrumbsBuilder === null) {
-            $this->breadcrumbsBuilder = new BreadcrumbsBuilder();
+            $this->breadcrumbsBuilder = new BreadcrumbsBuilder($this->getConfigurationPool()->getContainer()->getParameter('sonata.admin.configuration.breadcrumbs'));
```

Comment:
The goal since I separated the breadcrumbs builder from the admin is to depend less and less from it, not more. That means that in the end, the breadcrumbs builder should be provided with more and more dependencies through its constructor. Granted, this piece of code is ugly, but will disappear with 4.0, and it's good to have all the ugly code in one place rather than also add a getter in the admin for this parameter, then remove it in 4.0, don't you think?